source/text_handler.cpp

```diff
@@ -120,7 +120,8 @@ spv_result_t getWord(spv_text text, spv_position position, std::string* word) {
         case '\n':
         case '\r':
           if (escaping || quoting) break;
-        // Fall through.
+          word->assign(text->str + start_index, text->str + position->index);
+          return SPV_SUCCESS;
```

Comment:
Would be a nice spot for `[[fallthrough]];`, though ubiquitous use of C++17 features is probably a ways off.

Reply:
We could use a macro, guarded by compiler or C++ version.  But it seemed overkill in this case.